accord-core/src/main/java/accord/coordinate/ExecuteSyncPoint.java

```diff
@@ -195,6 +257,11 @@ protected void onSuccess()
         trySuccess(syncPoint);
     }
 
+    protected void sendApply(Node.Id to)
```

Comment:
this appears to be dead code, is this used by C*?

Reply:
I don't see it used in https://github.com/apache/cassandra/pull/3698

Reply:
probably my mistake, I meant to extract and call it directly. will fix

Reply:
yep, forgot to invoke it instead of CoordinateSyncPoint.sendApply